versioned_docs/version-1.3.0/install-kubeslice/kubeslice-cli/topology-configuration.mdx

```diff
@@ -50,8 +50,9 @@ The following table describes the helm chart configuration parameters.
 
 | Parameter                                                                 | Parameter Type | Description                                                                                    | Required  |
 | ------------------------------------------------------------------------- | -------------- | ---------------------------------------------------------------------------------------------- | --------- |
-| repo_alias                                                                | String         | The alias of the helm repo for KubeSlice charts.                                               | Optional  |
-| repo_url                                                                  | String         | The URL of the helm charts for KubeSlice.                                                      | Mandatory |
+| repo_alias                                                                | String         | The alias of the helm repo for KubeSlice charts. To use local chart provide the path to the charts.  | Optional  |
```

Comment:
Add a comma after `To use local chart,` 

Is it `To use a local chart`?

Reply:
I can change it to local helm chart repositories

Reply:
done